.github/workflows/test-library.yml

```diff
@@ -356,4 +561,98 @@ jobs:
       run: >-
         print("All's good")
       shell: python
+
+  publish-pypi:
+    name: Publish 🐍📦 ${{ needs.pre-setup.outputs.git-tag }} to PyPI
+    needs:
+    - check
+    - pre-setup  # transitive, for accessing settings
+    if: >-
+      fromJSON(needs.pre-setup.outputs.release-requested)
+    runs-on: Ubuntu-latest
+
+    environment:
+      name: release
+      url: >-
+        https://pypi.org/project/proxy.py/${{
+          needs.pre-setup.outputs.dist-version
+        }}
+
+    steps:
+    - name: Download all the dists
+      uses: actions/download-artifact@v2
+      with:
+        name: python-package-distributions
+        path: dist/
+    - name: >-
+        Publish 🐍📦 v${{ needs.pre-setup.outputs.git-tag }} to PyPI
+      uses: pypa/gh-action-pypi-publish@release/v1
+      with:
+        password: ${{ secrets.PYPI_TOKEN }}
+
+
+  publish-testpypi:
+    name: Publish 🐍📦 to TestPyPI
+    needs:
+    - check
+    - pre-setup  # transitive, for accessing settings
+    if: >-
+      false &&  # FIXME: remove this line once credentials are recovered
```

Comment:
@abhinavsingh I've added this to allow you to merge this as is. Once you recover the account and set up the secret, you'll be able to re-enable this job by simply dropping this line.